pygmo/test.py

```diff
@@ -1079,6 +1079,17 @@ def runTest(self):
         self.assertTrue(int(pop.get_x()[0][0]) != pop.get_x()[0][0])
         self.assertTrue(int(pop.get_x()[0][1]) != pop.get_x()[0][1])
 
+class wfg_test_case(_ut.TestCase):
+    """Test case for the UDP wfg
+        
+        """
+    
+    def runTest(self):
+        from .core import wfg, population
+        udp = wfg(prob_id=1, dim_dvs=5, dim_obj=3, dim_k=4)
+        pop = population(udp, 20)
+        self.assertTrue(prob.get_nx() == 5)
+        self.assertTrue(prob.get_nobj() == 3)
```

Comment:
In addition to writing the test case, you must also ensure it is actually run (see the ``suite.addTest()`` lines towards the end of the file).